test/testers.jl

```diff
@@ -153,6 +153,10 @@ end
             return x, fsymtest_pullback
         end
 
+        @testset "frule_test" begin
```

Comment:
Should we change the top testset name to mention that we are testing the `ignore` featue?
https://github.com/JuliaDiff/ChainRulesTestUtils.jl/pull/55/files#diff-3c133782d8380a9a5ea9ae765f612585R146